packages/nuxt/package.json

```diff
@@ -67,12 +67,12 @@
     "@clerk/shared": "workspace:^",
     "@clerk/types": "workspace:^",
     "@clerk/vue": "workspace:^",
-    "@nuxt/kit": "^3.14.159",
-    "@nuxt/schema": "^3.14.159",
+    "@nuxt/kit": "^3.16.0",
+    "@nuxt/schema": "^3.16.0",
```

Comment:
We need to update [`@nuxt/kit` to 3.16.0](https://github.com/nuxt/nuxt/pull/31079) to support proper type augmentation for `event.context.auth` in Nitro. 

This is a non-breaking change as it only requires the updated `@nuxt/kit` package and works with any Nuxt version.